samples/modules/tflite-micro/tflm_ethosu/README.rst

```diff
@@ -42,31 +42,28 @@ Quick Start
 
 .. code-block:: bash
 
-    west build -b alif_e7_dk/ae722f80f55d5xx0/rtss_he \\
-      alif/samples/modules/tflite-micro/tflm_ethosu/ -p always -- \\
-      -DETHOSU_TARGET_NPU_CONFIG=ethos-u55-128 \\
-      -DEXTRA_DTC_OVERLAY_FILE="boards/enable_ethosu55.overlay" \\
-      -G"Unix Makefiles"
+    west build -b alif_e7_dk/ae722f80f55d5xx0/rtss_he \
+      alif/samples/modules/tflite-micro/tflm_ethosu/ \
+      -DETHOSU_TARGET_NPU_CONFIG=ethos-u55-128 \
+      -DEXTRA_DTC_OVERLAY_FILE="boards/enable_ethosu55.overlay"
```

Comment:
```suggestion
    west build -b alif_e7_dk/ae722f80f55d5xx0/rtss_he \
      -DETHOSU_TARGET_NPU_CONFIG=ethos-u55-128 \
      -DEXTRA_DTC_OVERLAY_FILE="boards/enable_ethosu55.overlay"
```

Reply:
The same instructions applies for all the other build command. If you are reading these instructions you have already located the sample. No need to repeat same instructions the User Guide already provides.